data/import/glossary.csv

```diff
@@ -1915,6 +1917,8 @@ multi_cart_widget.cart.updated.success,Cart updated successfully,en_US
 multi_cart_widget.cart.updated.success,Einkaufswagen wurde erfolgreich aktualisiert,de_DE
 multi_cart_widget.cart.updated.error,Cart has been restricted or deleted,en_US
 multi_cart_widget.cart.updated.error,Warenkorb wurde eingeschränkt oder gelöscht,de_DE
+multi_cart_widget.cart.delete.error,Cart cannot be removed,en_US
+multi_cart_widget.cart.delete.error,Der Wagen kann nicht entfernt werden,de_DE
```

Comment:
Wagen => Warenkorb
This has to be fixed on nonsplit side afterword.

Reply:
Also Einkaufswagen in line 1917

Reply:
Bug ticket created https://spryker.atlassian.net/browse/CC-9948